main.rb

```diff
@@ -0,0 +1,76 @@
+require_relative './app'
+
+def run_option
+  puts 'Welcome to the Library App!'
+  puts 'Please choose an option:'
+  puts '1 - List all books.'
+  puts '2 - List all people'
+  puts '3 - Create a person'
+  puts '4 - Create a book'
+  puts '5 - Create a rental'
+  puts '6 - List all rentals for a given person ID'
+  puts '7 - Quit'
+end
+
+def create_person(app)
+  puts 'Do you want to create a student(1) or a teacher(2)? [Input the number]:-'
+  option = gets.chomp
+  parent_permission = true
+  puts 'age:-'
+  age = gets.chomp.to_i
+  puts 'name:-'
+  name = gets.chomp
+  puts 'Has parent permission?[Y/N]'
+  val = gets.chomp
+  parent_permission = false if %w[N n NO no No].include?(val)
+
+  case option.to_i
+  when 1
+    app.create_student(age, name, parent_permission)
+  when 2
+    puts 'specialization:-'
+    specialization = gets.chomp
+    app.create_teacher(age, specialization, name, parent_permission)
+  else
+    puts 'Invalid input ):'
+  end
+end
+
+def handel_option(option, app)
+  case option
+  when 3
+    create_person(app)
```

Comment:
- [ ] I cannot create a student or teacher either. Kindly fix this issue.

  ![fffirir](https://github.com/Tracy-miranja/school_library/assets/54848363/9ddc695a-fb2a-47fe-af3f-8108520102cf)
  ![ffifofo](https://github.com/Tracy-miranja/school_library/assets/54848363/9d30369a-a936-4371-bdfb-5276a879572d)

Reply:
Needs attention!